examples/with-algolia-react-instantsearch/app/page.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import Search from '../components/Search'
+
+export const dynamic = 'force-dynamic'
```

Comment:
Is this needed?

Reply:
This might not be needed for all search use cases, but it is necessary when users implement personalized search.

Reply:
Why is that? How is this library doing data fetching?